components/cards/default/card.hbs

```diff
@@ -0,0 +1,19 @@
+<div class="section--bg-white demo-section">
+  <div class="content-block">
+    <img
```

Comment:
I'm debating if we wouldn't be smart to do a <picture> > <source> setup on this image. I think for now let's stick with a plain <img>, but be open to swapping it out if the need arises.